jsoc/archive.md

```diff
@@ -2,19 +2,30 @@
 
 # Summer of Code Archives
 
-Julia has greatly benefited from the Google Summer of Code. In the last 6 years, over 70+ students have contributed to Julia and various Julia packages. Many of these students have become regular Julia contributors well beyond their summer projects. Take a look at the [Ideas Page](/jsoc) and contribute your ideas.
+Julia has greatly benefited from the Google Summer of Code. In the last 8 years, over 110+ students have contributed to Julia and various Julia packages. Many of these students, after their summer projects, have gone on to become regular contributors to Julia, a programming language that has been gaining much popularity in recent years.
+Take a look to our [Julia Seasons of Contributions Page](/jsoc) and contribute your ideas.
 
-## GSoC 2020
+## GSoC 2022
 
-In 2020, we are grateful to Google and the GSoC program for funding [16 awesome  students working on Julia projects](https://summerofcode.withgoogle.com/archive/2020/organizations/6163109162516480/#projects). In addition to the funding from Google, we [funded an additional 9 students](https://discourse.julialang.org/t/julia-seasons-of-contributions-to-supplement-gsoc-2020/38754) for the summer. 2020 was also the first year of the MLH Fellowship where we had 11 students working on Julia projects across the ecosystem. You can find out more about the 2020 Julia Season of Contributions [on our blog](https://julialang.org/blog/2020/09/gsoc-2020-wrapup/). 
+In 2022, we are delighted to recognize Google for its support and funding, as well as our Contributor. Nineteen talented and driven contributors worked on projects, making a meaningful impact. Check out the [Projects List](https://summerofcode.withgoogle.com/archive/2022/organizations/the-julia-language#projects-list) on the GSoC site for a look into what was done in 2022.
 
+For the 2022 idea list, [View Idea List](https://julialang.org/jsoc/projects/).
```

Comment:
```suggestion
For potential 2023 projects, check out the updated [idea List](https://julialang.org/jsoc/projects/).
```

Reply:
For 2023, can we update the JSoC page where we add, new notices and idea lists?